jenkins-pipelines/manager/manager-upgrade.jenkinsfile

```diff
@@ -0,0 +1,19 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    params: params,
+    manager: true,
+    backend: 'aws',
+    aws_region: 'us-east-1',
+
+    new_scylla_mgmt_repo: '',
+    scylla_repo_m: 'https://s3.amazonaws.com/downloads.scylladb.com/deb/ubuntu/scylla-2019.1-bionic.list',
```

Comment:
why is scylla_repo_m defaults to a Ubuntu 18 repo file?

Reply:
Mistake on my side, removed and added defaults of manager server and agent repos with manager 2.0 as the value. Fixed!